community/modules/file-system/DDN-EXAScaler/main.tf

```diff
@@ -42,7 +42,7 @@ locals {
 }
 
 module "ddn_exascaler" {
-  source          = "github.com/DDNStorage/exascaler-cloud-terraform//gcp?ref=a3355d50deebe45c0556b45bd599059b7c06988d"
+  source          = "github.com/DDNStorage/exascaler-cloud-terraform//gcp?ref=fafd923"
```

Comment:
nit: is there a reason to use the commit hash instead of the tagged version? Could we at least put the tagged version in a comment at the end of the line so we don't have to cross reference in the future:
```
...cloud-terraform//gcp?ref=fafd923"  # tag=scripts/x.x
```

Reply:
I had trouble when I tried using `ref=scripts/2.2.0`. It didn't recognize that there was more after the `/` and I couldn't get it to work, so I decided to stick to the commit hash for now. I added the comment for the tag, so it's easy to find the reference in the future.